README.md

```diff
@@ -33,6 +33,13 @@ Release builds are available on the [Releases page](https://github.com/nethermin
 3. To run the launcher: `nethermind`
 4. To run the runner: `nethermind -c mainnet`
 
+#### On Windows using Windows Package Manager
+
+1. `winget install nethermind` (running as administrator recommended)
+2. To run the launcher: `nethermind.launcher.exe`
+3. To run the runner: `nethermind.runner.exe -c mainnet` \
+   Or simply `nethermind -c mainnet` if step 1 has been run as administrator
```

Comment:
Trying to run this but it won't let me. Did you try this already? 

```
PS C:\Users\Falco> nethermind -c mainnet
2023-03-02 14-50-14.8357|Nethermind starting initialization.
2023-03-02 14-50-14.8861|Loading embedded plugins
2023-03-02 14-50-14.8864|  Found plugin type Nethermind.Consensus.AuRa.AuRaPlugin
2023-03-02 14-50-14.8865|  Found plugin type Nethermind.Consensus.Clique.CliquePlugin
2023-03-02 14-50-14.8867|  Found plugin type Nethermind.Consensus.Ethash.EthashPlugin
2023-03-02 14-50-14.8868|  Found plugin type Nethermind.Consensus.Ethash.NethDevPlugin
2023-03-02 14-50-14.8869|  Found plugin type Nethermind.Hive.HivePlugin
2023-03-02 14-50-14.8870|  Found plugin type Nethermind.UPnP.Plugin.UPnPPlugin
Resolved executing directory as C:\Users\Falco\AppData\Local\Microsoft\WinGet\Links.
2023-03-02 14-50-14.9036|Plugin assemblies folder C:\Users\Falco\AppData\Local\Microsoft\WinGet\Links\plugins was not found. Skipping.
2023-03-02 14-50-14.9816|Loading standard NLog.config file from C:\Users\Falco\AppData\Local\Microsoft\WinGet\Links\NLog.config.
```

Installing as administrator also won't let me run `nethermind.launcher.exe` or `nethermind.runner.exe` anymore.

Reply:
Everything works fine for me. Please DM me to discuss your issue.